### YamlMime:ManagedReference
items:
- uid: System.IdentityModel.SignatureVerificationFailedException
  id: SignatureVerificationFailedException
  children:
  - System.IdentityModel.SignatureVerificationFailedException.#ctor
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException
  fullName: System.IdentityModel.SignatureVerificationFailedException
  type: Class
  summary: "A kivétellel, hogy hiba esetén egy aláírás feldolgozása közben történt"
  syntax:
    content: 'public class SignatureVerificationFailedException : System.IdentityModel.Tokens.SecurityTokenException'
  inheritance:
  - System.Exception
  - System.SystemException
  - System.IdentityModel.Tokens.SecurityTokenException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  id: '#ctor'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> osztály."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonság egy rendszer által megadott üzenetet, amely leírja a hibát az új példányának.</xref:System.Exception.Message%2A> Ez az üzenet figyelembe veszi a rendszer jelenlegi kulturális környezethez.       A következő táblázat az új példány kezdeti tulajdonságértéket.      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Az üzenet honosított hibakarakterlánc. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException ();
    parameters: []
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> osztály egy adott hiba miatt."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonság új kivétel használva a `message` paraméter.</xref:System.Exception.Message%2A> A tartalma `message` célja, hogy ember által is értelmezhető. Ez a konstruktor védőfalkezelőbe annak érdekében szükség, hogy ez a karakterlánc a jelenlegi rendszer kulturális környezethez tartozó honosított.       A következő táblázat az új példány kezdeti tulajdonságértéket.      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| A üzenetet hiba karakterlánca. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A hibaüzenet, amely ismerteti a kivétel az az oka."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> a szerializált adatok osztályt."
  remarks: "Ez a konstruktor nevezik meglévő kölcsönökre az adatfolyam továbbított kivétel objektum deszerializálása közben."
  syntax:
    content: protected SignatureVerificationFailedException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> objektum, amely a szerializált objektum adatokat tartalmaz."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>objektum, amely a cél- és környezeti információkat tartalmaz.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IdentityModel.SignatureVerificationFailedException
  langs:
  - csharp
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  type: Constructor
  assemblies:
  - System.IdentityModel
  namespace: System.IdentityModel
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IdentityModel.SignatureVerificationFailedException&quot;> </xref> megadott hibaüzenet, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt, a osztályt."
  remarks: "Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság a konstruktornak átadott ugyanazt az értéket adja vissza.</xref:System.Exception.InnerException%2A>       A következő táblázat az új példány kezdeti tulajdonságértéket.      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| A belső kivétel referencia. |}   | <xref:System.Exception.Message%2A>| A üzenetet hiba karakterlánca. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public SignatureVerificationFailedException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "A hibaüzenet, amely ismerteti a kivétel az az oka."
    - id: inner
      type: System.Exception
      description: "A <xref:System.Exception>, amely a jelenlegi kivétel okát.</xref:System.Exception> Ha a `inner` paraméter nem <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a jelenlegi kivétel a következik be egy <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blokk, amely kezeli a belső kivételben találhatók."
  overload: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.IdentityModel.Tokens.SecurityTokenException
  isExternal: false
  name: System.IdentityModel.Tokens.SecurityTokenException
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException()
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException()
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException()
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor(System.String,System.Exception)
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException(String,Exception)
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
  fullName: System.IdentityModel.SignatureVerificationFailedException.SignatureVerificationFailedException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.IdentityModel.SignatureVerificationFailedException.#ctor*
  parent: System.IdentityModel.SignatureVerificationFailedException
  isExternal: false
  name: SignatureVerificationFailedException
  nameWithType: SignatureVerificationFailedException.SignatureVerificationFailedException
